extensions/amp-form/amp-form.md

```diff
@@ -196,6 +201,17 @@ You can configure your analytics to send these events as in the example below.
 </amp-analytics>
 ```
 
+`amp-form-submit` is fired before a request is sent. For this event, a set of variables that reflects your form's fields is generated. For an example, consider the following form:
```

Comment:
I don't have reservations since the publishers send the data to themselves. @rudygalfi do you have an issue with this?